### PR TITLE
Specify non-empty-array for Assert::uniqueValues

### PR DESCRIPTION
**Pull request by herndlm**
This was basically the only simple thing still missing. Implemented the same way as the `non-empty-string` + something (e.g. Assert::email) were implemented. 

See https://github.com/webmozarts/assert?tab=readme-ov-file#type-assertions
> Check that the given array contains unique values

### CODE REVIEW

File: src/Type/WebMozartAssert/AssertTypeSpecifyingExtension.php [hunk 1]
```diff
@@ -785,6 +785,12 @@ private function getExpressionResolvers(): array
 				};
 			}
 
+			foreach (['uniqueValues'] as $name) {
```
Comment (ondrejmirtes) [inline src/Type/WebMozartAssert/AssertTypeSpecifyingExtension.php]: Please write more simple code assigning the key, not a foreach.

**Review by ondrejmirtes — CHANGES_REQUESTED** [review]

File: src/Type/WebMozartAssert/AssertTypeSpecifyingExtension.php (hunk 1 above)
Comment (herndlm) [reply src/Type/WebMozartAssert/AssertTypeSpecifyingExtension.php]: sure, adapted. sorry, my laziness was on new heights there. it's way simpler now :)

**Review by ondrejmirtes — APPROVED** [review]

**Comment by ondrejmirtes** [comment]
Thank you.

**Comment by VincentLanglet** [comment]
Looking at the implementation, 
https://github.com/webmozarts/assert/blob/11cb2199493b2f8a3b53e7f19068fc6aac760991/src/Assert.php#L751
```
$a = [];
Assert::uniqueValues($a);
```

Seems valid, so I don't think we can assert the array is not empty after this assertion.

**Pull request merged**